download.py

```diff
@@ -1,12 +1,14 @@
 import logging
 import os
+import subprocess
+from pathlib import Path
 
 import requests
 
 from study_info.check_result import check_loaded_run, md5_checksum
 
 
-def download_run_fasterq_dump(accession, term, total_spots, out):
+def download_run_fasterq_dump(accession, term, total_spots, output_directory, *, core_count):
```

Comment:
With the addition of `core_count` this function now accepts 5 arguments, and it would make sense to pass some (or all) of them by name for readability. I made `core_count` argument keyword-only, since it's logically different from other arguments, and because it's not an argument of other similar functions. I didn't make other arguments keyword-only because it would require significantly more code changes.